src/sentry/issues/producer.py

```diff
@@ -64,9 +64,8 @@ def produce_occurrence_to_kafka(
     if payload_data is None:
         return
 
-    partition_key = None
-    if occurrence and occurrence.fingerprint:
-        partition_key = occurrence.fingerprint[0].encode()
+    fingerprint = payload_data["fingerprint"]
+    partition_key = fingerprint[0].encode()
```

Comment:
@aliu3ntry should we check if fingerprints is at least 1 item?

Reply:
should i emit some kind of log/metric if empty?

Reply:
@aliu3ntry that and prevent it from blowing up :)

Reply:
Tried a different approach that should throw an error in this case before producing.. let me know how it looks